### ISSUE TITLE
Problem with calling callbacks in TinyMcePluginLoader::outputTemplate

### ISSUE THREAD

**Issue by jacob418** [cliffparnitzky/TinyMcePluginLoader#10]
Im not shure if it is a general Issue, but in my case this Line (`TinyMcePluginLoader/system/modules/TinyMcePluginLoader/classes/TinyMcePluginLoader.php:89`) :
``` php
$arrTinyConfig = $this->$callback[0]->$callback[1]($arrTinyConfig);
```
caused this error:
```Fatal error: Uncaught exception Error with message Function name must be a string thrown in system/modules/TinyMcePluginLoader/classes/TinyMcePluginLoader.php on line 98
#0 system/modules/core/classes/BackendTemplate.php(115): TinyMcePluginLoader\TinyMcePluginLoader->outputTemplate('...', 'be_main')
#1 system/modules/core/controllers/BackendMain.php(275): Contao\BackendTemplate->output()
#2 system/modules/core/controllers/BackendMain.php(135): Contao\BackendMain->output()
#3 contao/main.php(20): Contao\BackendMain->run()
#4 {main}```

I was able to fix it this way:
``` php
$str1 = $callback[0] ;
$str2 = $callback[1] ;
$arrTinyConfig = $this->$str1->$str2($arrTinyConfig);
```

I hope this input will be helpfoul :)

**Comment by cliffparnitzky** [comment]
Hmmm, when i look into the Contao 3.5 core code, this seems to be right:

`$arrTinyConfig = $this->{$callback[0]}->{$callback[1]}($arrTinyConfig);`

Please test this in your environment.

**Comment by jacob418** [comment]
Yes i have... i guess thats the point.

**Comment by jacob418** [comment]
> `$arrTinyConfig = $this->{$callback[0]}->{$callback[1]}($arrTinyConfig);`

Is working fine for me.

**Comment by cliffparnitzky** [comment]
Fixed in 254cf3875ceae1f6bfefde41b72b0a422388e70a .

Please install version 2.1.6 (https://github.com/cliffparnitzky/TinyMcePluginLoader/releases/tag/2.1.6)

@jacob418 Thanks for reporting the bug.
